controller:
  buttons:
    manual_mode_select: 0
  axes:
    linear_speed_index: 4
    angular_speed_index: 3
  dead_zone: 3000
teleop:
  max_linear_speed: 4
  max_angular_speed: 4
